### DISCUSSION TITLE
Unexpected space before sidebars

### DISCUSSION THREAD

**Discussion by peterpeterparker** [facebook/docusaurus#9111] [Q&A] [answered]
Hi, any idea what can lead my website to have unexpected empty spaces in the sidebars?

For example: [https://juno.build/docs/intro](https://juno.build/docs/intro)

<img width="1536" alt="Capture d’écran 2023-06-29 à 14 30 41" src="https://github.com/facebook/docusaurus/assets/16886711/37ad22ce-fe78-4c7e-8cc3-22319ecc1a4e">

**Comment by Josh-Cena** [comment, marked as answer]
This is the padding that's used to prevent the navbar from obstructing the content. For example, on the Docusaurus site, when you scroll down and the navbar gets hidden, you will see that the sidebar and navbar both have empty spaces at the top. However, normally the sidebar container should extend all the way to the top of the screen, so that the top padding exactly overlaps with the navbar height, while in your case the container only extends to the bottom of the navbar. Do you have any custom CSS that can cause that?

**Reply by peterpeterparker** [reply]
Indeed a custom CSS breaks it!!!

```
.main-wrapper {
  position: relative;
  overflow-x: hidden; // <----- this particularly
}
```

Thanks a lot for the quick answer 🙏

**Reply by Josh-Cena** [reply]
That's much faster than I could have figured this out ^^
